tests/Microsoft.DotNet.XHarness.iOS.Tests/AppRunnerTests.cs

```diff
@@ -274,7 +274,6 @@ public async Task RunOnSimulatorSuccessfullyTest()
             _listener.Verify(x => x.Initialize(), Times.AtLeastOnce);
             _listener.Verify(x => x.StartAsync(), Times.AtLeastOnce);
             // _listener.Verify(x => x.Cancel(), Times.AtLeastOnce); // TODO: https://github.com/dotnet/xharness/issues/73
-            _listener.Verify(x => x.Dispose(), Times.AtLeastOnce);
```

Comment:
Why was this removed? should the app runner be disposing the listener?

Reply:
A mistake - bad revert, will fix